RunningOrder/Story/Views/NewStoryView+Logic.swift

```diff
@@ -32,7 +32,13 @@ extension NewStoryView {
         func createStory() {
             guard areAllFieldsFilled else { return }
 
-            let newStory = Story(sprintId: sprintId, name: name, ticketReference: ticketID, epic: epic)
+            let newStory = Story(
+                sprintId: sprintId,
+                name: name,
+                ticketReference: ticketID,
+                epic: epic,
+                creatorReference: nil
```

Comment:
why is it nil. ?

Reply:
good question. this is because this reference is set by CloudKit when receiving the story. I don't have any of this when I create the story locally. BUT, as I don't try to add this reference my self to the CKRecord when sending the data, I could create a fake one with the good info in it :) On the other hand, when we create it, we send it to CloudKit almost directly, and we receive the updated record that contains the data and it replace the created locally one, so It's not really useful

Reply:
alright thanks for explaining, maybe you could add a small comment for anyone who's having the. same question

Reply:
okay !